lizzy_client/lizzy.py

```diff
@@ -130,6 +130,18 @@ def get_traffic(self, stack_id: str, region: Optional[str]=None) -> dict:
         response.raise_for_status()
         return response.json()
 
+    def scale(self, stack_id: str, new_scale: int,
+              region: Optional[str]=None):
+        url = self.stacks_url / stack_id
+        data = {"new_scale": new_scale}
+        if region:
+            data['region'] = region
+
+        header = make_header(self.access_token)
+        response = url.patch(json=data, headers=header, verify=False)
```

Comment:
Please explain why `verify=False` in a comment, because it looks very suspicious.